files/var/notebooks/Storage.ipynb

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Let's call this file "Help".
I would like to add more useful info later, like "How long is the notebook session alive" etc.

Reply:
Ok, will update now

Reply:
or `README.ipynb`

Reply:
Agree, README is more expected here.